.changeset/soft-humans-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[chore] simplify internal prerender API
```

Comment:
It doesn't sound like this should actually need a changeset, right? It's just an internal refactoring.

Reply:
I was debating. I figured if anything broke on a version with this update it'd be easier to track it down with the changeset, but users should not need to care about it otherwise

Reply:
We don't normally add changesets for internal-only changes — are you saying we should start requiring a changeset for every PR?

Reply:
I wasn't sure if we had a policy around it. I'll drop this file